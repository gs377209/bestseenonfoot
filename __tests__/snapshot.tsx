import { render } from "@testing-library/react";
import Intro from "../src/components/intro";

it("renders homepage unchanged", () => {
  const { container } = render(<Intro />);
  expect(container).toMatchSnapshot();
});
