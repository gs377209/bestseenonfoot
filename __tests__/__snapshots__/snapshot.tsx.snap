// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <section
    class="mb-16 mt-16 flex flex-col items-center md:mb-12 "
  >
    <h1
      class="text-5xl font-bold leading-tight tracking-tighter md:pr-8 md:text-7xl"
    >
      Best Seen On Foot
    </h1>
    <h2
      class="mt-5 text-center text-4xl md:pl-8 md:text-left"
    >
      Welcome to our travels!
    </h2>
  </section>
</div>
`;
